iac-extensions/docker/src/test/java/org/sonar/iac/docker/tree/impl/FromTreeImplTest.java

```diff
@@ -19,13 +19,18 @@
  */
 package org.sonar.iac.docker.tree.impl;
 
+import java.util.List;
 import org.junit.jupiter.api.Test;
 import org.sonar.iac.docker.parser.grammar.DockerLexicalGrammar;
 import org.sonar.iac.docker.parser.utils.Assertions;
 import org.sonar.iac.docker.tree.api.AliasTree;
 import org.sonar.iac.docker.tree.api.DockerTree;
+import org.sonar.iac.docker.tree.api.ExposeTree;
+import org.sonar.iac.docker.tree.api.FileTree;
 import org.sonar.iac.docker.tree.api.FromTree;
 import org.sonar.iac.docker.tree.api.ImageTree;
+import org.sonar.iac.docker.tree.api.InstructionTree;
+import org.sonar.iac.docker.tree.api.MaintainerTree;
```

Comment:
why those changes?

Reply:
Mmhh probably related to my first version, those import are not used, I'm surprised it doesn't raise any code smell.
I will remove them, thanks 👍